src/ExternalImage.php

```diff
@@ -221,6 +230,33 @@ public function path(): string
         return URLHelper::get_rel_path($this->file);
     }
 
+    /**
+     * Gets the relative path to the uploads folder of an attachment.
```

Comment:
If it’s not an attachment and live in a theme folder, will this still be relative path to the uploads folder?